dev-packages/application-package/src/extension-package-collector.ts

```diff
@@ -60,7 +60,15 @@ export class ExtensionPackageCollector {
         }
         this.visited.set(name, true);
 
-        const packagePath = this.resolveModule(name + '/package.json');
+        let packagePath: string | undefined;
+        try {
+            packagePath = this.resolveModule(name + '/package.json');
+        } catch (error) {
+            console.log(`Failed to resolve module: ${name}`);
```

Comment:
I think it should be logged as a warning or error

Reply:
turned into a warning.